guides/client/js-interop.md

```diff
@@ -313,3 +319,42 @@ Hooks.Chart = {
 ```
 
 *Note*: In case a LiveView pushes events and renders content, `handleEvent` callbacks are invoked after the page is updated. Therefore, if the LiveView redirects at the same time it pushes events, callbacks won't be invoked on the old page's elements. Callbacks would be invoked on the redirected page's newly mounted hook elements.
+
+
+### Connecting multiple liveSockets
```

Comment:
I wouldn't document this as prominently. I think the documentation for the liveSocket option is enough.